model/proto/model.proto

```diff
@@ -145,6 +145,9 @@ message Span {
     (gogoproto.customname) = "ProcessID"
   ];
   repeated string warnings = 12;
+  bool incomplete = 13 [
```

Comment:
please add documentation for the new field (even though the rest of the file doesn't, but it's TBD)